release-team/role-handbooks/documentation-guides/update-release-docs-new.md

```diff
@@ -1,88 +1,180 @@
-*   Update the OSS feature tracking spreadsheet with progress on reviews and merge status for each feature doc, e.g. [Kubernetes Features OSS tracking board (1.7 release)](https://docs.google.com/spreadsheets/d/1IJSTd3MHorwUt8i492GQaKKuAFsZppauT4v1LJ91WHY/edit?usp=sharing).
-*   Attend Kubernetes Tactics meetings.
-*   Attend Kubernetes Burndown meetings.
-*   Sync with the release manager biweekly, then weekly, then daily.
-
-### Work with the engineers and community
-
-*   Sync with the dev managers and SIG leads so that they know to direct the engineers to you if they have any docs questions.
-*   Send announcements to:
-    *   [gke-kubernetes-team@google.com](mailto:gke-kubernetes-team@google.com)
-    *   [kubernetes-sig-release@google.com](mailto:kubernetes-sig-release@google.com), Slack #sig-release
-    *   [kubernetes-sig-leads@google.com](mailto:kubernetes-sig-leads@google.com)
-    *   feature owners listed in the OSS feature tracking spreadsheet
-*   Announcements should cover:
-    *   TL;DR on the docs release process with your contact info.
-    *   Dates for:
-        *   filling in features in the OSS feature tracking spreadsheet
-        *   opening a placeholder PR in the k8s.io repo
-        *   when PRs should be completed and ready for review
-        *   when PRs tech reviews should be completed
-        *   when review feedback should be incorporated by and Docs and Tech LGTMs given
-        *   the drop-dead merge date and time for any last minute PRs
-*   Meet with engineers early about feature docs and give guidance on what should be a Concept, Task, Tutorial, or Reference doc.
-*   Recruit community tech writers to help with docs reviews.
+*   Kubernetes [#sig-release channel](https://kubernetes.slack.com/messages/C2C40FMNF/) on Slack
 
-### Generate reference docs
+## Start the release cycle
+
+Release cycles usually begin 1-2 weeks after the previous cycle completes.
+
+### General starting tasks
+
+1. Get connected.
+
+- Join the Kubernetes [#sig-release Slack channel](https://kubernetes.slack.com/messages/C2C40FMNF/).
+
+    Introduce yourself as the docs lead for the release.
+
+- Connect with the release manager.
+
+- Join these Google groups:
+
+    - [kubernetes-sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+    - [kubernetes-sig-leads](https://groups.google.com/forum/#!forum/kubernetes-sig-leads)
+    - [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev)
+
+- Make sure you're included in the release team docs, for example: [release 1.11](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.11/release_team.md).
+
+- Review (and correct if necessary) the proposed timeline for the release, for example: [release 1.11](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.11/release-1.11.md).
```

Comment:
is this the right landing page for this bullet? FWIW we are not using .md anymore and exclusively using only the README